### PR TITLE
Redesigned a bit

### PR DESCRIPTION
**Pull request by bartekpacia**
The only person who cares about is (maybe) you, Diogo. And you know what happened here, so I'm not gonna write any description

### CODE REVIEW

**Comment by diogoscf** [comment]
I like the description

**Comment by diogoscf** [comment]
Tests are failing

**Comment by bartekpacia** [comment]
yeah i forgot to update them

File: src/app.ts [hunk 1]
```diff
@@ -1,21 +1,75 @@
...
-}
+admin.initializeApp()
 
-if (discordRunning) {
-  const discordBot = new discord.EnforcingDiscordBot(new DiscordConfig())
-  console.log("Started Discord bot.")
-}
+const TELEGRAM_TOKEN_NAME = "projects/telegram-lang-enforcer/secrets/TELEGRAM_TOKEN/versions/latest"
+const TELEGRAM_TOKEN_NAME_DEV = "projects/telegram-lang-enforcer/secrets/TELEGRAM_TOKEN_DEV/versions/latest"
+const DISCORD_TOKEN_NAME = "projects/telegram-lang-enforcer/secrets/DISCORD_TOKEN/versions/latest"
```
Comment (diogoscf) [inline src/app.ts:20]: This is a bit hardcoded

File: src/app.ts [hunk 2]
```diff
@@ -1,21 +1,75 @@
...
 
-const telegramRunning = process.argv.includes("--telegram")
-const discordRunning = process.argv.includes("--discord")
+export const config = new CoreConfig()
 
-if (telegramRunning) {
-  const telegramBot = new telegram.EnforcingTelegramBot(new TelegramConfig())
-  console.log("Started Telegram bot.")
-}
+admin.initializeApp()
```
Comment (diogoscf) [inline src/app.ts:16]: Wait how does it get the credentials now?

Comment (bartekpacia) [reply src/app.ts:16]: idk but works

Comment (diogoscf) [reply src/app.ts:16]: lol

Comment (bartekpacia) [reply src/app.ts]: oke brought it back, just in case

Comment (diogoscf) [reply src/app.ts]: alr

File: src/app.ts (hunk 1 above)
Comment (bartekpacia) [reply src/app.ts:26]: they we'll tell people to name their projects with this name in the GCP console.

BTW I'm fed up with this GCP secret manager bullshit and will likely remove it from `app.ts` and leave good, old, plain env vars

Comment (diogoscf) [reply src/app.ts:26]: Ah, a sigh of relief

**Pull request merged**